addons/web/static/src/views/fields/properties/property_value.js

```diff
@@ -149,6 +149,10 @@ export class PropertyValue extends Component {
 
         if (this.props.type === "many2one" && value && value.length === 2) {
             return formatMany2one(value);
+        } else if (this.props.type === "integer") {
+            return value === undefined || value === false ? "0" : formatInteger(value);
```

Comment:
can we do something like this ? (same bellow)
```suggestion
            return formatInteger(value || "0");
```